web/api/backend.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"time"
+	jwtmiddleware "github.com/auth0/go-jwt-middleware"
+	"github.com/dgrijalva/jwt-go"
+)
+
+// Credentials object read body from the request body
+type Credentials struct {
+	RequestType string
+	Args        map[string]interface{}
+	UseCache    string
+	Body        map[string]interface{}
```

Comment:
Perhaps `UseCache` could be a boolean here. Also, for `Body`, if we name it `UploadedCredentials`, it may be more explicit as to what is supposed to be stored there.

Reply:
Renamed